### ISSUE TITLE
convert() documentation missing import

### ISSUE THREAD

**Issue by bc0n** [JuliaLang/julia#50273]
Implicit conversion of custom types requires `import Base.convert` but the documentation does not mention this or provide a useful example.  Propose adding this mwe to the [documentation](https://github.com/JuliaLang/julia/blob/master/doc/src/manual/conversion-and-promotion.md):

```
module Test
  import Base.convert
  type MyType <: Number
    n :: Int
  end
  convert(::Type{Int}, x::MyType) = x.n
end
```

Julia Version 1.8.1
Commit afb6c60d69 (2022-09-06 15:09 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 12 × AMD Ryzen 5 1600X Six-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-13.0.1 (ORCJIT, znver1)
  Threads: 1 on 12 virtual cores
Environment:
  JULIA_PKG_DEVDIR = W:\...\julia
  JULIA_EDITOR = code
  JULIA_NUM_THREADS =

Installed prebuilt package directly downloaded from julialang.org

**Comment by brenhinkeller** [comment]
So this is true for any base function you want to extend (always need to import first), but I suppose more examples never hurt

**Comment by bc0n** [comment]
Yeah @brenhinkeller, the point is that high-traffic documentation should be atomic.  Providing convert() is one of the first things a user needs to do when integrating new structs with external libraries, and they're usually informed of this not by reading the entire manual but by an error message.
